src/System.Diagnostics.Debug/tests/DebuggerTests.cs

```diff
@@ -11,19 +11,35 @@ public class DebuggerTests
         [Fact]
         public void IsAttached()
         {
-            // TODO: Implement this when Debugger is properly implemented
+            bool b = Debugger.IsAttached;
         }
 
         [Fact]
-        public void Launch()
+        public void Break()
         {
-            // TODO: Implement this when Debugger is properly implemented
+            if (!Debugger.IsAttached)
+                Debugger.Break();
```

Comment:
I meant it like this. I don't want to mess with a Debugger if one is *actually* attached. I only want to call these functions if they won't mess something up.